main_window/main.py

```diff
@@ -268,3 +263,13 @@ def mapResizedEvent(self, event) -> None:
         :type event:
         """
         self.MappingThread.setDesiredMapSize(event.width, event.height)
+
+
+window = None
+
+
+def start(*args, **kwargs):  # TODO: does this need its own file??
```

Comment:
```suggestion
def start(*args, **kwargs):
```

This comment was only relevant when it was in the start.py file. It was just questioning whether it was even necessary. But now I think the start file actually has a good purpose.

Reply:
Good catch! We certainly don't want any more TODOs lying around.